src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.fxml

```diff
@@ -30,8 +30,13 @@
           <Insets left="20.0" />
        </VBox.margin>
       </VBox>
-      <CheckBox fx:id="cleanUpBiblatex" text="%Convert to biblatex format (e.g., store publication date in date field)" />
-      <CheckBox fx:id="cleanUpBibtex" text="%Convert to BibTeX format (e.g., store publication date in year and month fields)" />
+     <RadioButton text="%Convert to biblatex format (e.g., store publication date in date field)">
+      <toggleGroup>
+        <ToggleGroup fx:id="conversionToggleGroup" />
+       </toggleGroup>
+   </RadioButton>
```

Comment:
Indent seems tob e off. Opening and matching closing tags should be on the same column.